app/models/pipeline_run.rb

```diff
@@ -1444,19 +1444,26 @@ def get_summary_contig_counts(min_contig_size)
   end
 
   def get_summary_contig_counts_v2(min_contig_size)
-    summary_dict = {} # key: count_type:taxid , value: contigs, contig_reads
-    contig_lineages(min_contig_size).each do |c|
-      lineage = JSON.parse(c.lineage_json)
-      lineage.each do |count_type, taxid_arr|
-        taxids = taxid_arr[0..1]
-        taxids.each do |taxid|
-          summary_per_tax = summary_dict[taxid] ||= {}
-          summary_per_tax_and_type = summary_per_tax[count_type.downcase] ||= {}
-          summary_per_tax_and_type[c.read_count] ||= 0
-          summary_per_tax_and_type[c.read_count] += 1
+    # key: taxid:count_type, value: read_counts: count
```

Comment:
You probably didn't write this comment, but just noting that I can't figure out what this means.
I think a better comment would be:
```
Stores the number of contigs that match a given taxid, count_type (nt or nr), or read_count (number of reads aligned to that contig.
```

Reply:
Done! Thanks for the clear phrasing 😄